src/renderer/components/dock/dock.tsx

```diff
@@ -64,6 +70,14 @@ class NonInjectedDock extends React.Component<Props & Dependencies> {
       closeTab(selectedTab.id);
       this.element?.current.focus();  // Avoid loosing focus when closing tab
     }
+
+    if(ctrlKey && code === "Period") {
+      this.nextTab(Direction.next);
+    }
+
+    if(ctrlKey && code === "Comma") {
+      this.nextTab(Direction.prev);
```

Comment:
_nit:_ probably having `Direction` as enum only for that purpose is redundant and something like `this.nextTab(-1)` should be enough.